### PR TITLE
Add version matrix

### PR DESCRIPTION
**Pull request by arturkow2**

### CODE REVIEW

**Comment by BeataZdunczyk** [comment]
@arturkow2 great, thank you.  QEMU is missing from the list.

File: docs/variants/versions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Version matrix
+
+## About
+
+The table below contains an up to date list of all supported platforms,
+including latest supported coreboot and EDK2 versions and branches from which
+you can build firmware for your platform.
+
+| Platform                | coreboot branch                                                                                       | Dasharo base                                                          | coreboot version | EDK2 version | EDK2 rev                                                                                    |
```
Comment (krystian-hebel) [inline docs/variants/versions.md:9]: I think that `coreboot version` should be removed. It is strictly correlated with `Dasharo base`, and dropping this column will make more space for the rest.

**Review by krystian-hebel — CHANGES_REQUESTED** [review]

File: docs/variants/versions.md (hunk 1 above)
Comment (arturkow2) [reply docs/variants/versions.md]: Done

**Review by krystian-hebel — APPROVED** [review]

**Comment by krystian-hebel** [comment]
Merging despite CI complaining about broken link to https://review.coreboot.org/plugins/gitiles/memtest86plus/+/0bd34c22, caused by https://osfw.slack.com/archives/CCWLQKEHG/p1725964231625859

**Comment by pietrushnic** [comment]
I saw the message on coreboot ml. Since there is no mirror, the only solution would be to remove the link if gitles will not be brought back. What would you say about mirroring that ourselves?

**Pull request merged**